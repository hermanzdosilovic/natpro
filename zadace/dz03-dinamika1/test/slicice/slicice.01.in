10 6 2
9
1
1
2
8
10
3
3
7
4
