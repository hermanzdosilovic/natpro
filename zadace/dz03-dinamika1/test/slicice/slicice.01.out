mirko
mirko
mirko
mirko
mirko
slavko
slavko
slavko
slavko
mirko
